exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,23 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  if (number < 0) {
+    return "true";
+  } else {
+    number > 0;
```

Comment:
This does not cause any issues in the code completing, but does nothing. Did you mean to assign the result of `number > 0;` to another variable?

Reply:
Removed code that served no purpose.